networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -112,6 +147,80 @@ TEST_F(DynBetweennessGTest, runDynApproxBetweennessSmallGraphEdgeDeletion) {
 
 }
 
+TEST_P(DynBetweennessGTest, testDynApproxBetweenessGeneratedGraph) {
+    DorogovtsevMendesGenerator generator(200);
+    Graph G1 = generator.generate();
+    Graph G(G1, isWeighted(), isDirected());
+    count n = G.numberOfNodes();
+    if (isWeighted()) {
+        Aux::Random::setSeed(42, false);
+        G.forEdges([&](node u, node v) { G.setWeight(u, v, Aux::Random::probability()); });
+    }
+
+    DEBUG("Generated graph of dimension ", G.upperNodeIdBound());
+    double epsilon = 0.1; // error
+    double delta = 0.1; // confidence
+    DynApproxBetweenness dynbc(G, epsilon, delta);
+    Betweenness bc(G);
+    dynbc.run();
+    bc.run();
```

Comment:
We never use the original betweenness scores, so there is no reason to run the algorithm here.